### PR TITLE
Upgrade to oxipng 7.0.0

### PR DESCRIPTION
**Pull request by RaiaN**
Hey there!
I've upgrade `pyoxing` to support `oxipng` 7.0.0. All tests are passed succesfully.

### CODE REVIEW

**Comment by RaiaN** [comment]
Major changes:
- Zlib is removed (completely replaced with Libdeflate)
- `filter` is now `op::Rowfilter`

**Comment by nfrasser** [comment]
@RaiaN this is great, thanks for taking the time to do this! I'll review in detail within the next couple of days

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "pyoxipng"
-version = "6.0.0"
-edition = "2021"
+version = "7.0.0"
+edition = "2022
```
Comment (nfrasser) [inline Cargo.toml:4]: Missing quote here, also there is no rust 2021

```suggestion
edition = "2021"
```

**Review by nfrasser — APPROVED** [review]
Looks good, going to merge and make a few tweaks. Thanks again @RaiaN!

**Pull request merged**